middle_end/flambda2/from_lambda/closure_conversion.ml

```diff
@@ -327,9 +329,9 @@ module Inlining = struct
           | Known free_names -> free_names
         in
         let make_inlined_body =
-          make_inlined_body ~callee
+          make_inlined_body ~callee ~region_inlined_into
```

Comment:
I can't put a review comment there, but at line 250 you have the definition of this `make_inlined_body` function (which is a wrapper around `Inlining_helpers.make_inlined_body`). I think that it should also be patched to take the extra `my_region` and `region_inlined_into` parameters.

Reply:
Amusingly I think `make_inlined_body` is returning a partially-applied version of `Inlining_helpers.make_inlined_body` which is why it works at the moment, but I will eta expand.